packages/components/popover/src/PopoverContent/PopoverContent.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+
+export const getPopoverContentStyles = () => ({
+  container: css({
+    background: tokens.colorWhite,
+    border: 0,
+    borderRadius: tokens.borderRadiusMedium,
+    boxShadow: tokens.boxShadowDefault,
+    zIndex: tokens.zIndexDropdown,
+  }),
```

Comment:
I think this is where we should add padding

Reply:
I don't think we should touch the inner content styles, since it can be customised in a different ways. 
For example for such divider you will have to remove somehow those paddings or set a negative margins: 
![image](https://user-images.githubusercontent.com/10744462/130249664-b5483a5b-e6f9-4425-b4a9-8fc40b7827a9.png)

Reply:
I guess with this image we have two things `PopoverContent` and `PopoverContentFooter` (?????)

So we could make an API like this: 
```
<PopoverContent>
  <MenuItem />
  <MenuItem />
</PopoverContent>
<PopoverContentFooter>
  <TextLink/>
</PopoverContentFooter>  
```

If ask the users to deal with the spacing themselves, they would have to do this:
```
<PopoverContent>
  <Box margin="spacingM">
     <MenuItem />
     <MenuItem />
  <Box/>
  <Box margin="spacingM">
     <TextLink/>
  <Box/>
</PopoverContent>
```

But we won't be able to prevent people from things like not using any spacing, using a bigger or smaller spacing than `spacingM`
I also can see some of the FE in Contentful complaining that they need to pass a `Box` all the time hahahaha

Reply:
But maybe you agreed with the designers about making `Popover` flexible as you said in other comment
And this strict spacing that I'm talking about will come with `Menu` and the other components using `Popover`

then... just ignore me hahahaha

Reply:
> And this more strict spacing that I'm talking about will come with Menu and the other components using Popover
💯 Let's see what we will have with designs and with specific components in the end. And then adjust this `Popover`. For now it's hard to peak the right decision, I even don't have new popover, menu, autocomplete etc. designs yet. 🤷‍♂️

Reply:
My two cents, Popover is a utility component that shouldn't do any styling. Keep that in `Menu` and `Modal` and whatever else.

Reply:
Yep... I changed my mind hahahah 
Let's do the padding thing in `Menu` and the other components 👌